tests/kdc/check-bx509.in

```diff
@@ -573,6 +650,45 @@ if which jq >/dev/null; then
     fi
 fi
 
+echo "Fetch TGTs (batch, authz fail)"
+${kadmin} modify --max-ticket-life=10d krbtgt/${R}@${R}
+(set -vx; csr_grant pkinit bar@${R} foo@${R})
+${kdestroy}
+token=$(KRB5CCNAME=$cache2 $gsstoken HTTP@$server)
+if (set -vx;
+    curl -o "${cachefile}.json" -Lgsf                                   \
+         --resolve ${server}:${bx509port}:127.0.0.1                     \
+         -H "Authorization: Negotiate $token"                           \
+         "http://${server}:${bx509port}/get-tgts?cname=bar@${R}&cname=baz@${R}"); then
+    echo "Got TGTs with /get-tgts end-point that should have been denied"
+    exit 2
+fi
+
+echo "Fetch TGTs (batch, authz pass)"
```

Comment:
Check for 404s?  cnames that don't exist.

Reply:
Rather, check that we get 200 for authorized by non-existent names, and then check that the response body contains a JSON text with an error for the principal(s) that didn't exist.

Reply:
Fixed.